src/element-templates/Validator.js

```diff
@@ -266,7 +266,7 @@ export function getSchemaVersion(schemaUri) {
 export function filteredSchemaErrors(schemaErrors) {
   return filter(schemaErrors, (err) => {
     const {
-      dataPath,
+      instancePath,
```

Comment:
Whoops. This sweeped in as an undocumented breaking change with `v1.8.1` ([a patch release](https://github.com/bpmn-io/element-templates-validator/blob/main/CHANGELOG.md#181)).

Reply:
Should we roll back the changes and re-release as v2?

Reply:
Would be the clean approach, yes.

Reply:
I'll take care of it

Reply:
Could make sense to release v2.0.1 with [the additional changes](https://github.com/bpmn-io/element-templates-validator/commits/main/). Let's at least follow-up with a v2.0.0 release.

Reply:
> Could make sense to release v2.0.0 with [the additional changes](https://github.com/bpmn-io/element-templates-validator/commits/main/). Let's at least follow-up with a v2.0.0 release.

I released the latest changes as 2.0.0 and re-published 1.8.0 as 1.8.2, so the 1.8 branch is usable: https://github.com/bpmn-io/element-templates-validator/blob/main/CHANGELOG.md#200

Reply:
Also updated this PR to use v2

Reply:
Thanks. Great! :)